ArticleTemplates/collectionTemplate.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <title>Interactive</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="initial-scale=1, maximum-scale=1" />
+    <link rel="stylesheet" type="text/css" media="all" id="fontStyles" href="__TEMPLATES_DIRECTORY__assets/css/fonts-__PLATFORM__.css" />
+    <!--<link rel="stylesheet" type="text/css" media="all" href="__TEMPLATES_DIRECTORY__assets/css/interactive.css" />-->
```

Comment:
`interactive.css` exists because the interactive template has a very minimal set of styles applied to it.

All other templates use a combination of `__SYNC_STYLES__.css` and `fonts-__PLATFORM__.css` in the `<head>` and `__ASYNC_STYLES__` which are loaded by the JS. This template should be no different, so I'd just remove the commented out `link`.